contributing-guidelines.md

```diff
@@ -123,17 +125,17 @@ If you have opened the pull request early and know that its contents are not rea
 When you are happy with it and are happy for it to be merged into the main repository, change the "[WIP]" in the title of the pull request to "[Ready for review]".
 
 A member of _AutSPACEs_ community will then review your changes to confirm that they can be merged into the main repository.
-A [review][github-review] will probably consist of a few questions to help clarify the work you've done.
+A [review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews) will probably consist of a few questions to help clarify the work you've done.
 Keep an eye on your GitHub notifications and be prepared to join in that conversation.
 
-You can update your [fork][github-fork] of _AutSPACEs_ and the pull request will automatically update with those changes.
+You can update your fork of _AutSPACEs_ and the pull request will automatically update with those changes.
 You don't need to submit a new pull request when you make a change in response to a review.
 
 You can also submit pull requests to other contributors' branches!
-Do you see an open pull requeset that you find interesting and want to contribute to?
+Do you see an open pull request that you find interesting and want to contribute to?
 Simply make your edits on their files and open a pull request to their branch!
 
-GitHub has a [nice introduction][github-flow] to the pull request workflow, but please [get in touch](#get-in-touch) if you have any questions :balloon:.
+GitHub has a [nice introduction](https://docs.github.com/en/get-started/quickstart/github-flow) to the pull request workflow, but please [get in touch](#get-in-touch) if you have any questions :balloon:.
```

Comment:
There doesn't seem to actually be a 'get-in-touch' section. I'd say that should be for another issue/PR, but just flagging it.